tests/Bridge/Laravel/Console/Commands/SearchIndexLiveCommandTest.php

```diff
@@ -0,0 +1,91 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests\LoyaltyCorp\Search\Bridge\Laravel\Console\Commands;
+
+use Illuminate\Container\Container;
+use Illuminate\Contracts\Container\Container as ContainerInterface;
+use LoyaltyCorp\Search\Bridge\Laravel\Console\Commands\SearchIndexLiveCommand;
+use LoyaltyCorp\Search\Interfaces\IndexerInterface;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\NullOutput;
+use Symfony\Component\Console\Output\OutputInterface;
+use Tests\LoyaltyCorp\Search\Stubs\Handlers\HandlerStub;
+use Tests\LoyaltyCorp\Search\Stubs\IndexerStub;
+use Tests\LoyaltyCorp\Search\TestCase;
+
+/**
+ * @covers \LoyaltyCorp\Search\Bridge\Laravel\Console\Commands\SearchIndexCommand
+ * @covers \LoyaltyCorp\Search\Bridge\Laravel\Console\Commands\SearchIndexLiveCommand
+ *
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects) Required for thorough testing
+ */
+class SearchIndexLiveCommandTest extends TestCase
+{
+    /**
+     * Ensure search handlers are indeed passed to the indexSwap method
+     *
+     * @return void
+     *
+     * @throws \ReflectionException
+     */
+    public function testSearchHandlersPassedToIndexSwapMethod(): void
+    {
+        $indexer = new IndexerStub();
+        $container = new Container();
+        $container->tag([
+            HandlerStub::class
+        ], ['search_handler']);
+        $command = $this->createInstance([], new NullOutput(), $indexer, $container);
+
+        $command->handle();
+
+        self::assertSame(1, $indexer->getIndicesSwapped());
```

Comment:
As above, I'd prefer this command to swap things atomically and we'll need to test 2 or 3 indexes together

Reply:
@merk the alias remove/add happens atomically, but on a per-index basis - not all possible indices at once.
Are you saying it's preferred to do all indices alias switchover atomically, not atomically one index at a time?

Reply:
Yeah, its better to send a single request with all index changes required, otherwise you might end up with frontend making requests against the wrong indexes during the swapover